segment.go

```diff
@@ -121,27 +121,27 @@ func newBlockAndHeader() interface{} {
 	}
 }
 
-// NewReader creates a new segment reader.
+// newReader creates a new segment reader.
 // You can call Next to get the next chunk data,
 // and io.EOF will be returned when there is no data.
-func (seg *segment) NewReader() *segmentReader {
```

Comment:
Please split this into a single PR
For better review.

Reply:
done